recipe/meta.yaml

```
@@ -31,9 +32,9 @@ test:
     - freezegun >=0.2.8
     - pretend
     - python-rapidjson  # [not s390x]
+    - pytest-asyncio
```

```suggestion
    - pytest-asyncio >=0.17
```

is `simplejson` replaced by `python-rapidjson`?

and `pytest >=6.0`